.travis.yml

```diff
@@ -58,6 +58,19 @@ matrix:
     - php: 7.0
       env: DRIVER_VERSION=devel SERVER_VERSION=3.4
       addons: *common_addons
+    - php: 7.0
+      env: DRIVER_VERSION=devel SERVER_VERSION=3.4
+      addons: *common_addons
+      before_script:
+        - mongod --version
+        - pecl install -f mongodb-${DRIVER_VERSION}
+        - php --ri mongodb
+        - composer install --dev --no-interaction --prefer-source
+        - ulimit -c
+        - ulimit -c unlimited -S
+        - composer require --dev phpstan/phpstan
+      script:
+        - vendor/bin/phpstan analyse -l 0 src
```

Comment:
I assume the intention here is to only run PHPStan for a single job, since it would be redundant to run it otherwise?

Rather than duplicate the `before_script` block, it may be preferable to add `phpstan/phpstan` as a development dependency in `composer.json` (as we do with PHPUnit). Then, would only have to overwrite the `script` block here.

Note that as of https://github.com/mongodb/mongo-php-library/pull/481, I'm now launching mongo-orchestration and a MongoDB server in `before_script` and terminating it in `after_script`. That certainly isn't needed for PHPStan to perform its static analysis.

Would using [build stages](https://blog.travis-ci.com/2017-09-12-build-stages-order-and-conditions), as [Doctrine ORM does](https://github.com/doctrine/doctrine2/blob/bcc9da2f4a7a534ec1ffa58da6ae199efc1fc10a/.travis.yml#L69), help here?